application-services/custom/camera-management/appcamera/app.go

```diff
@@ -37,23 +42,31 @@ func NewCameraManagementApp(service interfaces.ApplicationService) *CameraManage
 
 func (app *CameraManagementApp) Run() error {
 	if err := app.service.LoadCustomConfig(app.config, "AppCustom"); err != nil {
-		return errors.Wrap(err, "failed to load custom configuration")
+		return errors.NewCommonEdgeX(errors.KindServerError, "failed to load custom configuration", err)
+	}
+
+	var err error
+	for i := 0; i < maxRetries; i++ {
+		app.lc.Infof("Querying EVAM pipeline statuses.")
+		if err = app.queryAllPipelineStatuses(); err != nil {
+			app.lc.Errorf("Unable to query EVAM pipeline statuses. Is EVAM running? %s", err.Error())
+			time.Sleep(time.Second)
+		} else {
+			break // no error, so lets continue
+		}
+	}
+	if err != nil {
+		app.lc.Errorf("Unable to query EVAM pipeline statuses after %d tries. .")
```

Comment:
You need to put in the value of `i` for this error message.

Reply:
Also there is an extra period in the message.